InfiniteRecharge/src/main/java/frc/util/PbVictorSpx.java

```diff
@@ -0,0 +1,17 @@
+package frc.util;
+
+import com.ctre.phoenix.motorcontrol.can.WPI_VictorSPX;
+
+import edu.wpi.first.wpilibj.SpeedController;
+
+public class PbVictorSpx extends WPI_VictorSPX implements SpeedController {
+
+    public PbVictorSpx(int deviceNumber) {
+        super(deviceNumber);
+    }
+
+    public void set(double power) {
+        this.set(power);
```

Comment:
should be `super.set(power)`

Reply:
Done. What is the difference?

Reply:
`this` refers to the class `PbVictorSpx` ("this" class), `super` refers to the superclass it is extending (`WPI_VictorSPX`)
if `this.set()` calls `this.set()` (itself) it won't actually do anything!

Reply:
Oh ok